examples/printerdemo_mcu/zephyr/src/slint-zephyr.cpp

```diff
@@ -380,20 +447,26 @@ void zephyr_process_input_event(struct input_event *event)
             button = slint::PointerEventButton::Left;
             slint::invoke_from_event_loop([=, button = button.value()] {
                 __ASSERT(ZEPHYR_WINDOW, "Expected ZephyrWindowAdapter");
-                ZEPHYR_WINDOW->window().dispatch_pointer_move_event(pos);
-                ZEPHYR_WINDOW->window().dispatch_pointer_press_event(pos, button);
+                // Transform the physical screen position to the rendered position
```

Comment:
To me "rendered" is the screen position.
Maybe logical coordinate is a better term.

Reply:
Done.